src/Opdex.Platform.Infrastructure/Clients/CirrusFullNodeApi/Modules/BlockStoreModule.cs

```diff
@@ -31,16 +31,16 @@ public Task<BlockReceiptDto> GetBlockAsync(Sha256 blockHash, CancellationToken c
 
         using (_logger.BeginScope(logDetails))
         {
-            return GetAsync<BlockReceiptDto>(uri, cancellationToken);
+            return await GetAsync<BlockReceiptDto>(uri, cancellationToken: cancellationToken);
         }
     }
 
-    public Task<Sha256> GetBestBlockAsync(CancellationToken cancellationToken)
+    public async Task<Sha256> GetBestBlockAsync(CancellationToken cancellationToken)
     {
-        return GetAsync<Sha256>(CirrusUriHelper.Consensus.GetBestBlockHash, cancellationToken);
+        return await GetAsync<Sha256>(CirrusUriHelper.Consensus.GetBestBlockHash, cancellationToken: cancellationToken);
```

Comment:
Doesn't matter but `async` and `await` can be left out if the awaited task is returning directly. Not a nit or something to fix more of an fyi incase you were unaware 💪

Reply:
The addition of `async`/`await` is to adhere to these recommendations https://blog.stephencleary.com/2016/12/eliding-async-await.html

It's more of a nit but the example shown in the article makes the difference pretty clear 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/39023939/155755880-009b285e-8696-4068-8b24-fb81d08bb533.png">

I've updated it here because the `ApiClientBase` will throw

Reply:
The addition of `async`/`await` is to adhere to these recommendations https://blog.stephencleary.com/2016/12/eliding-async-await.html

It's more of a nit but the example shown in the article makes the difference pretty clear 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/39023939/155755880-009b285e-8696-4068-8b24-fb81d08bb533.png">

I've updated it here because the `ApiClientBase` will throw

Reply:
The addition of `async`/`await` is to adhere to these recommendations https://blog.stephencleary.com/2016/12/eliding-async-await.html

It's more of a nit but the example shown in the article makes the difference pretty clear 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/39023939/155755880-009b285e-8696-4068-8b24-fb81d08bb533.png">

I've updated it here because the `ApiClientBase` will throw